docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,189 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/s/<space_id>/api/cases/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>, depending on the
+`owner` of the cases you're seeking.
+
+=== Path parameters
+
+`<space_id>`::
+(Optional, string) An identifier for the space. If it is not specified, the
+default space is used.
+
+=== Query parameters
+
+`defaultSearchOperator`::
+(Optional, string) The default operator to use for the `simple_query_string`.
+
+`fields`::
+(Optional, array of strings) The fields in the entity to return in the response.
```

Comment:
> The fields in the entity to return in the response.

I copied this description from https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/common/api/cases/case.ts#L136, however, its intent is still unclear to me.

If I enter a single value (e.g. "fields=id"), I get an error:
> {"statusCode":400,"error":"Bad Request","message":"Invalid value \"id\" supplied to \"fields\""}

If I enter two values (e.g. "fields=id&fields=title), I get results but they don't seem to be any different than what I get when I omit these query parameters. Am I using it wrong?

Reply:
Could you please try `fields=[id]`?

Reply:
in that case, i get: 
>curl: (3) bad range in URL position 48:
https://localhost:5601/api/cases/_find?fields=[id]
                                                                               ^
                                                                               
I tried adding multiple values and quotation marks to no avail. I must be missing something obvious

Reply:
This is a bug from our side. I will create a PR to fix it. Thanks!

Reply:
Thanks!  For now, I've omitted that parameter from the docs via 39609bae03faa53aea1b75f13578ac9617b53844

Reply:
PR with the fix: https://github.com/elastic/kibana/pull/128143

Reply:
With the fix lined up, I think you should add the `fields` parameter back into the docs

Reply:
Thanks!  I'll actually merge this PR as-is and add the fields parameter docs in the other PR.